### PR TITLE
Add "clean" script

### PR DESCRIPTION
**Pull request by swashcap**
* **Problem:** Stale stuff is included in builds
* **Solution:** Add a `clean` script and run it pre-`build`
* **Testing:** _???_

This was originally a PR to remove all the source code and project root stuff, but [the electron-build docs](https://www.electron.build/configuration/contents) don't make config easy to understand. How do we ignore `.tsx?`?!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "checkup",
+  "description": "health check anything, monitor statuses with visual indicators in your status bar",
```
Comment (swashcap) [inline package.json]: electron-builder gave a "missing description" warning. Might as well add it?

File: scripts/clean.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```
Comment (swashcap) [inline scripts/clean.sh]: Shell scriptin’ _FTW_

File: scripts/clean.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+set -e
+
+rm -rf dist_web &
+find src -type f \( -name '*.js' -o -name '*.js.map' \) \
+  -not -name 'configure.template.js' \
+  -exec rm {} \+ &
+
+wait
```
Comment (cdaringe) [inline scripts/clean.sh]: I'd really prefer no bash, personally.

File: scripts/clean.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  try {
+    const globs = await Promise.all(
+      ['.cache', 'dist', 'dist_web', '{scripts,src,test}/**/*.{js,js.map}'].map(
+        p => glb(path.resolve(__dirname, '..', p))
+      )
+    )
+
+    const removed = await Promise.all(
+      globs
+        .flat()
```
Comment (swashcap) [inline scripts/clean.ts]: [`Array#flat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat): _nice_

**Pull request merged**